Sources/SwiftSyntax/generated/RenamedChildrenCompatibility.swift

```diff
@@ -1167,73 +1167,6 @@ extension ClassDeclSyntax {
   }
 }
 
-extension ClosureCaptureSyntax {
-  @available(*, deprecated, renamed: "unexpectedBetweenNameAndEqual")
-  public var unexpectedBetweenNameAndAssignToken: UnexpectedNodesSyntax? {
-    get {
-      return unexpectedBetweenNameAndEqual
-    }
-    set {
-      unexpectedBetweenNameAndEqual = newValue
-    }
-  }
-  
-  @available(*, deprecated, renamed: "equal")
-  public var assignToken: TokenSyntax? {
-    get {
-      return equal
-    }
-    set {
-      equal = newValue
-    }
-  }
-  
-  @available(*, deprecated, renamed: "unexpectedBetweenEqualAndExpression")
-  public var unexpectedBetweenAssignTokenAndExpression: UnexpectedNodesSyntax? {
-    get {
-      return unexpectedBetweenEqualAndExpression
-    }
-    set {
-      unexpectedBetweenEqualAndExpression = newValue
-    }
-  }
-  
-  @available(*, deprecated, renamed: "ClosureCaptureSyntax(leadingTrivia:_:specifier:_:name:_:equal:_:expression:_:trailingComma:_:trailingTrivia:)")
-  @_disfavoredOverload
-  public init(
-      leadingTrivia: Trivia? = nil,
-      _ unexpectedBeforeSpecifier: UnexpectedNodesSyntax? = nil,
-      specifier: ClosureCaptureSpecifierSyntax? = nil,
-      _ unexpectedBetweenSpecifierAndName: UnexpectedNodesSyntax? = nil,
-      name: TokenSyntax? = nil,
-      _ unexpectedBetweenNameAndAssignToken: UnexpectedNodesSyntax? = nil,
-      assignToken: TokenSyntax? = nil,
-      _ unexpectedBetweenAssignTokenAndExpression: UnexpectedNodesSyntax? = nil,
-      expression: some ExprSyntaxProtocol,
-      _ unexpectedBetweenExpressionAndTrailingComma: UnexpectedNodesSyntax? = nil,
-      trailingComma: TokenSyntax? = nil,
-      _ unexpectedAfterTrailingComma: UnexpectedNodesSyntax? = nil,
-      trailingTrivia: Trivia? = nil
-    
-  ) {
-    self.init(
-        leadingTrivia: leadingTrivia, 
-        unexpectedBeforeSpecifier, 
-        specifier: specifier, 
-        unexpectedBetweenSpecifierAndName, 
-        name: name, 
-        unexpectedBetweenNameAndAssignToken, 
-        equal: assignToken, 
-        unexpectedBetweenAssignTokenAndExpression, 
-        expression: expression, 
-        unexpectedBetweenExpressionAndTrailingComma, 
-        trailingComma: trailingComma, 
-        unexpectedAfterTrailingComma, 
-        trailingTrivia: trailingTrivia
-      )
-  }
-}
-
```

Comment:
People might still be relying on these compatibility overloads, so we shouldn’t remove them.

Reply:
I added those to `SwiftSyntaxCompatibility.swift`.